### ISSUE TITLE
Not able to fork the jenkins repository

### ISSUE THREAD

**Issue by pushker001** [jenkins-infra/helpdesk#3278]
### Type(s) of account issue

I've been blocked by the anti-spam system

### Summary

 recently I have wanted to contribute to Jenkins that's why to ask the community MarkEWaite to provide me with plugins so I can contribute but I was unable to fork the repository. The same message is showing every time I try to fork the repository "you can't fork this repository at this time"
here you can see the messages
https://community.jenkins.io/t/regarding-beginner-friendly-issue/4677?u=pushker001

### Which URL are you trying to login at?

https://github.com/jenkinsci/built-on-column-plugin

**Comment by lemeurherve** [comment]
From [what I'm seing here](https://github.com/pushker001?tab=repositories), you've already forked the Jenkins repo: https://github.com/pushker001/jenkins, that's why you're seeing this error.

**Comment by pushker001** [comment]
so how can i fix this

**Comment by dduportal** [comment]
Hi @pushker001 , you might want to delete your fork, and fork again.

Follow these steps:
- Browse to the settings of your pre-existing fork: https://github.com/pushker001/jenkins/settings
- In the section "General", scroll down to the last section (in red) titled "Dangez Zone".
- Click on the "Delete Repository" button (last item) and follow instructions.
- Once deleted, you might want to wait 5 min and retry forking the jenkinsci/jenkins repository again.

Let us know

**Comment by daniel-beck** [comment]
>  One pull request submitted by @pushker001 was a spam pull request (changing space characters on a file in Jenkins core), so it seemed possible to me that the @pushker001 account might have been blocked.

Yup, I blocked the account for spam. https://github.com/jenkinsci/jenkins/pull/7441 is clearly nonsense.

**Comment by NotMyFault** [comment]
> Yup, I blocked the account for spam. [jenkinsci/jenkins#7441](https://github.com/jenkinsci/jenkins/pull/7441) is clearly nonsense.

Could you not do that in the future anymore, please? I see why you've blocked the user, but based on my prior experience with moderating (core) PRs, labeling them as spam and telling the submitter not to do that is more than enough.
If they'd legit spam PRs, create annoying comments or disrupt the overall atmosphere, that'd be a different case, of course.

It can be hard to get into open source development, and being blocked from further contributions, because you submitted something that is not up to our contribution guidelines, can be pretty frustrating and may alienate you at all.

**Comment by daniel-beck** [comment]
> something that is not up to our contribution guidelines

I've never blocked users for genuine, but (IMO) low quality contributions. I block users for vandalism and spam. In this case,

* The YAML file became newly syntactically invalid
* The documentation change is clearly unhelpful too

… and there's nothing else besides that.

I'm open to unbanning the user, but let's not pretend there was _any_ merit to the core PR.

**Comment by NotMyFault** [comment]
> but let's not pretend there was _any_ merit to the core PR

I didn't say that, and I do agree, spam and harmful content doesn't belong here.

But to be fair, I find your reaction a bit excessive, getting blocked after one PR with a few changes, whether of use for the project or not, sends off wrong signals :/
Everyone has to start somewhere, and sometimes it takes a few times before something useful comes out, especially with the complex codebase core is at.

Do you think we could get along with blocking users for repeatable/more severe offenses only, in the future?
I feel your stance, but maybe you get my concerns too :/

**Comment by pushker001** [comment]
Sorry but my intention was not spaming in the repository I am new here and really want to contribute in open source and that's why I come on Jenkins to contribute but I don't able to understand how I can contribute that why I try to explore in the file and remove radom spaces from the code and that leads spaming

**Comment by daniel-beck** [comment]
Given there are other participants in this conversation, I would welcome further opinions on unblocking @pushker001 from jenkinsci.

I think that is separate from the policy discussion that was triggered by this block.

👍 to unblock, 👎 to keep the block active perhaps?

**Comment by daniel-beck** [comment]
> But to be fair, I find your reaction a bit excessive, getting blocked after one PR with a few changes, whether of use for the project or not, sends off wrong signals :/

If the signal is no tolerance for pull requests that do not attempt to be a contribution, that's the correct signal. Being welcoming to newbies is one thing, tolerating non-contributions is another. Those are inconsiderate of current contributors' time, as well as their previous efforts in making the project accessible through contribution "getting started" guides and tutorials.

>  Everyone has to start somewhere

Find a typo or grammar problem in literally any file. Fix it. There, it's a genuine contribution that, at least in core, is even likely going to be merged. It's not difficult. We've probably had a dozen or so PRs for typos in comments in core's JS YUI debug resources alone.

> and sometimes it takes a few times before something useful comes out, especially with the complex codebase core is at.

As with what you wrote previously, this argument makes no sense in this conversation. This isn't even PR 7465, which looks like a good faith contribution (might want to get the keyword in). No useful contribution was even attempted, so "core is difficult" is not an argument against my position. I didn't block the submitter of 7410 and 7418; in fact I tried in multiple comments to explain the challenges with accomplishing their goal and showing a path forward.

**Comment by PierreBtz** [comment]
If I may, as a less frequent contributor in the project as some you, here are a few thoughts:

* While it's true that a community must be welcoming, it's also true that if said community put in places a lot of documentation to facilitate onboarding, it wouldn't be fair for regular contributors who invested time in documentation to lose time with spam either. In the real world you don't go to somebody's place and start breaking stuff just to see what it does...

* I have no problem seeing spam trigger some kind of reaction, but I feel this should be documented, like one spam PR you are warned, you start again 7 days blocked, again bye bye.

In this specific case, I'll go +1 since I have a feeling the message was understood. That being said, and in order not to be in the same situation again @pushker001:

>  I try to explore in the file and remove radom spaces from the code 

You changed a yaml file used to configure a Github action. Yaml obeys to a specific syntax, there are a lot of tutorials on the format available ([here for instance the first answer from Google](https://www.cloudbees.com/blog/yaml-tutorial-everything-you-need-get-started)).
You can read more about Github actions [here](https://docs.github.com/en/actions).
As a general rule of thumb, all non plain text files obey to syntax rules and it's recommended to understand those rules before changing them, your preferred IDE usually can help you to verify the syntax too.

**Comment by daniel-beck** [comment]
Thanks for your feedback everyone. Given it has been unanimous so far, I've went ahead and unblocked @pushker001, and .

I still think the conversation around what it means to be a welcoming community and what we can expect of newcomers is still one we should have, if not here, then elsewhere.

**Comment by dduportal** [comment]
Hello @pushker001 , can you confirm that you can now fork successfully from jenkinsci organization?

**Comment by lemeurherve** [comment]
Looking at recent comments from @pushker001 at https://community.jenkins.io/t/regarding-beginner-friendly-issue/4677/30 he seems to have been able to fork the repository, closing.
